CHANGES.md

```diff
@@ -1,5 +1,30 @@
 ## In the next release
 
+## 0.1.1
+* Trace negotiated SNI & ALPN at the end of the server handshake.
```

Comment:
Would love to see a brief prose summary of this release before the listicle.

Reply:
WDYT of 90033aa404f663c7dd186e12b8e8a286324ed7be?

Reply:
LGTM, thanks!